### PR TITLE
[FEAT] Creates a button to add SFL token to MetaMask on successful withdraw of SFL token(s).

### LINKED ISSUE #828: [FEAT] Import SFL token address to Metamask.
Once a user withdraws SFL, it would be good to add the token address into their Metamask. This will make it easier for them to see the SFL that has been transfer.

**Includes**

Once SFL is withdrawn, inside of the success modal include a button to add the token to your wallet.

1. User Withdraws SFL
2. Success modal shows
3. User clicks button to add SFL token address to Metamask

**Technical Details**

See this API - https://ethereum.stackexchange.com/questions/99343/how-to-automatically-add-a-custom-token-to-metamask-with-ethers-js

### PR DESCRIPTION
**Pull request by newb23**
# Description

Creates a button to add SFL token to MetaMask on successful withdraw of SFL token(s).
Closes #828

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Local with `yarn test`. Also had to override the `error` state in GoblinLand.tsx to display the modal for testing.
Test Suites: 12 passed, 12 total
Tests:       103 passed, 103 total
Snapshots:   0 total
Time:        24.33 s
Ran all test suites.
Done in 
25.42s.
firefox_Vqebi0QC51
https://user-images.githubusercontent.com/7065115/167292215-5871805e-beb4-4984-93c4-641bc2d170ba.mp4

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/components/Withdrawn.tsx [hunk 1]
```diff
@@ -44,7 +64,7 @@ export const Withdrawn: React.FC = () => {
           PolygonScan
         </a>
       </span>
-
+      <Button onClick={addTokenToMetamask}>Add SFL to MetaMask</Button>
```
Comment (Pranav-yadav) [inline src/features/game/components/Withdrawn.tsx:67]: add vertical margin `my-2` as suited..

File: src/features/game/components/Withdrawn.tsx [hunk 2]
```diff
@@ -44,7 +64,7 @@ export const Withdrawn: React.FC = () => {
...
       <Button onClick={() => goblinService.send("REFRESH")}>Ok</Button>
```
Comment (Pranav-yadav) [inline src/features/game/components/Withdrawn.tsx:68]: add margin bottom `mb-2` or `mb-3` as suited..

**Review by Pranav-yadav — CHANGES_REQUESTED** [review]

**Comment by baron** [comment]
Shouldn't the token value be referenced from VITE_TOKEN_CONTRACT ?

**Comment by baron** [comment]
Shouldn't the token contract address be referenced from TOKEN_CONTRACT?

**Pull request merged**